components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -129,7 +145,12 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
             <ListItemText
                 primary={title}
                 className={combine('listItemText')}
-                secondary={getSubtitle()}
+                secondary={
+                    <>
+                        <Typography variant={'body2'}>{getSubtitle()}</Typography>
+                        <Typography variant={'body2'}>{getInfo()}</Typography>
+                    </>
```

Comment:
Looks fine. If you don't pass in either a subtitle or info then they don't render, which does mean, however, that if you use the info prop without a subtitle then it will look just like the subtitle